packages/rn-tester-e2e/README.md

```diff
@@ -0,0 +1,129 @@
+# RNTester E2E folder
+
+In this folder we have a local setup for doing E2E testing over RNTester via the usage of [Appium](https://appium.io/) and [WebDriverIO](https://webdriver.io/) and [Jest](https://jestjs.io/).
+
+- [Setting up](#setting-up)
+  - [(one-off) Setting up Appium](#one-off-setting-up-appium)
+  - [Building RNTester app](#building-rntester-app)
+    - [Building for iOS](#building-for-ios)
+    - [Building for Android](#building-for-android)
+  - [Setting up the RNTester E2E folder](#setting-up-the-rntester-e2e-folder)
+- [Testing the RNTester app E2E](#testing-the-rntester-app-e2e)
+- [Adding new tests (and project structure)](#adding-new-tests-and-project-structure)
+
+## Setting up
+
+### (one-off) Setting up Appium
+
+The first step you need to do is to ensure to install the tooling:
+
+```bash
+npm install appium@2.0.0-beta.71 -g
+appium driver install uiautomator2
+appium driver install xcuitest
+```
+
+> More details about drivers in Appium [here](https://appium.github.io/appium/docs/en/2.0/guides/managing-exts/) and [here](https://appium.github.io/appium/docs/en/2.0/quickstart/uiauto2-driver/)
+
+You should not need to run install commands for drivers separately more than once, even if you bump the dep in package.json.
+
+### Building RNTester app
+
+Building manually *.app* and *.apk* is required to run automation tests on local environment.
+
+0. *(optional)* If you previously built RNTester, you may need to clean up build files and Pods:
+
+    ```bash
+    yarn test-e2e-local-clean
```

Comment:
```suggestion
    yarn test-e2e-local-clean && yarn
```
Just to make sure that the dependencies are reinstalled! ;)

Reply:
Hi @kelset,
This is fixed in the last MR. Could you please resolve this comment?